src/app/public/modules/file-attachment/file-attachment.component.html

```diff
@@ -0,0 +1,69 @@
+<div
+  class="sky-single-file-wrapper">
+  <div
+    #labelWrapper
+    class="sky-file-drop-label-wrapper"
+    [ngClass]="{ 'sky-control-label-required': required && hasLabel() }">
+    <ng-container
+      *ngTemplateOutlet="tempOutlet" >
+    </ng-container>
+  </div>
+  <div
+    class="sky-single-file-upload sky-single-file-drop sky-single-file-drop-target"
+    [ngClass]="{ 'sky-single-file-drop-accept': acceptedOver, 'sky-single-file-drop-reject': rejectedOver }"
+    (dragover)="fileDragOver($event)"
+    (dragenter)="fileDragEnter($event)"
+    (dragleave)="fileDragLeave($event)"
+    (drop)="fileDrop($event)">
+    <input
+      tabindex="-1"
+      aria-hidden="true"
+      type="file"
+      class="sky-file-input-hidden"
+      (change)="fileChangeEvent($event)"
+      [attr.accept]="acceptedTypes ? acceptedTypes: null"
+      #fileInput>
```

Comment:
Code style > Watch your ordering of attributes here: https://host.nxt.blackbaud.com/skyux-team-docs/style#component-html-tags

Also note that the ending `>` should always be on the next line. I noticed that in several places.